src/service/core-services.js

```diff
@@ -141,4 +142,8 @@ function installAmpdocServicesInternal(ampdoc, isEmbedded) {
     : installStorageServiceForDoc(ampdoc);
   installGlobalNavigationHandlerForDoc(ampdoc);
   installGlobalSubmitListenerForDoc(ampdoc);
+  if (!isEmbedded) {
+    // Embeds do not show loading indicators.
```

Comment:
nit: can you either give the _why_ or remove the comment. as-is it doesn't tell you anything beyond what is obvious in the code :)